content/ko/docs/concepts/architecture/nodes.md

```diff
@@ -147,9 +147,9 @@ kubectl describe node <insert-node-name-here>
 {{< table caption = "노드 컨디션과 각 컨디션이 적용되는 시기에 대한 설명들이다." >}}
 | 노드 컨디션 | 설명 |
 |----------------|-------------|
-| `Ready`        | 노드가 상태 양호하며 파드를 수용할 준비가 되어 있는 경우 `True`, 노드의 상태가 불량하여 파드를 수용하지 못할 경우 `False`, 그리고 노드 컨트롤러가 마지막 `node-monitor-grace-period` (기본값 40 기간 동안 노드로부터 응답을 받지 못한 경우) `Unknown` |
-| `DiskPressure`    | 디스크 사이즈 상에 압박이 있는 경우, 즉 디스크 용량이 넉넉치 않은 경우 `True`, 반대의 경우 `False` |
-| `MemoryPressure`    | 노드 메모리 상에 압박이 있는 경우, 즉 노드 메모리가 넉넉치 않은 경우 `True`, 반대의 경우 `False` |
+| `Ready`        | 노드의 상태가 양호하며 파드를 수용할 준비가 되어 있는 경우 `True`, 노드의 상태가 불량하여 파드를 수용하지 못할 경우 `False`, 그리고 노드 컨트롤러가 마지막 `node-monitor-grace-period` (기본값 40 기간 동안 노드로부터 응답을 받지 못한 경우) `Unknown` |
+| `DiskPressure`    | 디스크 사이즈 상에 압박이 있는 경우, 즉 디스크 용량이 넉넉하지 않은 경우 `True`, 반대의 경우 `False` |
+| `MemoryPressure`    | 노드 메모리 상에 압박이 있는 경우, 즉 노드 메모리가 넉넉하지 않은 경우 `True`, 반대의 경우 `False` |
```

Comment:
```suggestion
| `DiskPressure`    | 디스크 사이즈 상에 압박이 있는 경우, 즉 디스크 용량이 충분하지 않은 경우 `True`, 반대의 경우 `False` |
| `MemoryPressure`    | 노드 메모리 상에 압박이 있는 경우, 즉 노드 메모리가 충분하지 않은 경우 `True`, 반대의 경우 `False` |
```

- '디스크 용량이 적은 경우', 또는 '충분하지 않은 경우'는 어떨까요?
> `True` if pressure exists on the disk size--that is, if the disk capacity is low; otherwise `False`